config/config-secrets-example.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  ##########################
+  #                        #
+  #   EXAMPLE CONFIG MAP   #
+  #                        #
+  ##########################
+  # This ConfigMap serves only as an example. It is named with the "-example"
+  # suffix so that we don't accidently stomp on ourselves with something like
+  # `ko apply`.
+  #
+  # Ideally, `kf install` is used to setup the config map. However this
+  # example can obviously be used to manually setup a ConfigMap.
+  #
+  # If you are using GKE and want to manually setup a secret do the following:
+  # kubectl -nkf create secret docker-registry knative-build-key \
+  #   --docker-username=_json_key \
+  #   --docker-server https://gcr.io \
+  #   --docker-password="$(cat /path/to/key.json)"
+  name: config-secrets-example
```

Comment:
So I actually left the -example suffix on purpose because I didn't want this file to overwrite existing configuration. Given an operator is going to have to setup the secret and the name of the secret.